modules/tensor_mechanics/test/tests/porosity/tests

```diff
@@ -16,4 +16,34 @@
       detail = 'using automatic differentiation materials.'
     []
   []
+  [negative]
+    requirement = 'The system shall handle a negative calculated porosity computed from a strain tensor'
+    [zero]
+      type = CSVDiff
+      input = 'negative_porosity_zero.i'
```

Comment:
Sorry, I just saw that I'm assigned to this, although it looks like @dschwen took it over. I'm not great at visually diffing 3 files, but I'm guessing that they're very close to identical. Would it make sense to use cli_args here? Otherwise, this looks good to me.

Reply:
Yeah...could use cli_args...it's always a battle to me when to use it or not, but decided not here...

Reply:
Changed to cli_args